infra/recipes/terraform/azure/environments/container/versions.tf

```diff
@@ -0,0 +1,12 @@
+terraform {
+    required_providers {
+        azurerm = {
+            source = "hashicorp/azurerm"
+            version = "< 3.90.0" 
```

Comment:
`Azurerm` provider seems to be duplicate as we have already defined in another place in this PR.

Reply:
Yes but the [shared versions file](https://github.com/linkedin/openhouse/blob/0e52fa812f5967f6788c8e271b883576e27ba07b/infra/recipes/terraform/azure/versions.tf) has another provider `docker` in it, which we do not need. Should I create two separate common-versions so we can reuse the azure provider but not the other ones?